src/MembershipContextFactory.php

```diff
@@ -48,11 +43,8 @@ public function __construct( array $config, Configuration $doctrineConfig ) {
 
 	public function newMappingDriver(): MappingDriverChain {
 		$driver = new MappingDriverChain();
-		// We're only calling this for the side effect of adding Mapping/Driver/DoctrineAnnotations.php
-		// to the AnnotationRegistry. When AnnotationRegistry is deprecated with Doctrine Annotations 2.0,
-		// instantiate AnnotationReader directly instead.
-		$driver->addDriver( $this->doctrineConfig->newDefaultAnnotationDriver( self::ENTITY_PATHS, false ), self::ENTITY_NAMESPACE );
-		$driver->addDriver( new XmlDriver( self::DOCTRINE_CLASS_MAPPING_DIRECTORY ), self::DOMAIN_ENTITY_NAMESPACE );
+		$driver->addDriver( new XmlDriver( self::DOCTRINE_CLASS_MAPPING_DIRECTORY ), self::ENTITY_NAMESPACE );
+		$driver->addDriver( new XmlDriver( self::DOMAIN_CLASS_MAPPING_DIRECTORY ), self::DOMAIN_ENTITY_NAMESPACE );
```

Comment:
I did not pay attention here - could you add a `visitMappingDriver` method here? Otherwise we will have a `MappingDriverChain` (from `newMappingDriver`) inside another `MappingDriverChain` (from the fundraising app `DoctrineFactory`).